spec/message_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'message'
+describe Message do
+  subject(:message) { described_class.new }
+  it 'creates an instance variable for @my_number' do
+    expect(subject.my_number).to eq('+447510060718')
```

Comment:
maybe this should be expect(message.my_number), seeing as you defined it that way in line 3